src/nvim/testdir/test39.in

```diff
@@ -88,6 +88,10 @@ bbbb
 cccc
 dddd
 
+yaaa
+���
+bbb
+
```

Comment:
These lines were probably added by mistake, they are not used or verified in the test. I propose leaving them in in order not to irritate those who are going to merge future patches or port the test to Lua (this had me rage for a good few minutes.)